### PR TITLE
Better template

### PR DESCRIPTION
**Pull request by Yajo**
Remove extra spaces, show license in ALT text, don't separate contributors by blank line.

### CODE REVIEW

**Comment by guewen** [comment]
:+1:

File: template/module/README.rst [hunk 1]
```diff
@@ -44,6 +44,7 @@ Contributors
 ------------
 
 * Firsname Lastname <email.address@example.org>
```
Comment (pedrobaeza) [inline template/module/README.rst]: Please take the occassion to fix this now to `Firstname`

File: template/module/README.rst [hunk 2]
```diff
@@ -44,6 +44,7 @@ Contributors
...
+* Second Guy <second.guy@example.org>
```
Comment (pedrobaeza) [inline template/module/README.rst]: I prefer the more neutral `Second person` (without gender)

**Comment by yvaucher** [comment]
:+1:

**Comment by max3903** [comment]
:+1:

**Pull request merged**